atomic_reactor/utils/koji.py

```diff
@@ -552,26 +551,9 @@ def get_rpms():
 
 def get_builder_image_id(build_id, osbs):
     """
-    Find out the docker ID of the buildroot image we are in.
+    Retrieve the docker ID of the buildroot image we are in from the environment.
     """
-    try:
-        buildroot_tag = os.environ["OPENSHIFT_CUSTOM_BUILD_BASE_IMAGE"]
-    except KeyError:
-        return ''
-
-    try:
-        pod = osbs.get_pod_for_build(build_id)
-        all_images = pod.get_container_image_ids()
-    except OsbsException as ex:
-        logger.error("unable to find image id: %s", ex)
-        return buildroot_tag
-
-    try:
-        return all_images[buildroot_tag]
-    except KeyError:
-        logger.error("Unable to determine buildroot image ID for %s",
-                     buildroot_tag)
-        return buildroot_tag
+    return os.environ.get('OPENSHIFT_CUSTOM_BUILD_BASE_IMAGE', '')
```

Comment:
explain in the commit, that what we would get from get_container_image_ids is already in that env variable, that is why we are doing it